### ISSUE TITLE
【question】Regarding the update of the latest assessment results issue

### ISSUE THREAD

**Issue by GuangchaoDeng** [erikbern/ann-benchmarks#565]
1. I saw from the official website that the latest results available are from 2023. I would like to ask if there are any plans to update the latest evaluation results?
2. If we want to submit an algorithm and update the latest results display, what do we need to provide?

**Comment by erikbern** [comment]
Yeah I'm planning to re-run all results, let me find time in the next few weeks!

**Comment by GuangchaoDeng** [comment]
What is the update frequency for the subsequent plans?
